components/brave_rewards/browser/rewards_service.h

```diff
@@ -183,6 +190,34 @@ class RewardsService : public KeyedService {
   virtual void GetConfirmationsHistory(
       brave_rewards::ConfirmationsHistoryCallback callback) = 0;
 
+  virtual void GetAutoContributeCount(
+      const rewards_counter::OnRewardsCountedCallback callback) = 0;
+
+  virtual void ClearAutoContributeData(
+      OnRemoveRewardsDataCallback callback,
+      uint64_t reconcile_stamp) = 0;
+
+  virtual void OnClearAutoContributeData(
+      OnRemoveRewardsDataCallback callback,
+      bool result) = 0;
+
+  virtual void RemoveRewardsData(
+      int remove_mask,
+      OnRemoveRewardsDataCallback callback) = 0;
+
+  virtual void OnRemoveRewardsData(
+      OnRemoveRewardsDataCallback callback,
+      bool result) = 0;
+
+  virtual void BroadcastReconcile(bool contribution_in_progress) = 0;
+
+  virtual void GetIsContributionInProgress(
+      OnGetContributionIsInProgressCallback callback) = 0;
+
+  virtual void OnGetIsContributionInProgress(
```

Comment:
why is this here? It's not part of the public api for RewardsService and should be a private method

Reply:
Removed